backend/compact-connect/tests/smoke/purchasing_privileges_smoke_tests.py

```
@@ -103,7 +103,7 @@ def test_purchase_privilege_options():
     logger.info('Successfully verified purchase privilege options')
 
 
-def test_purchasing_privilege():
+def test_purchasing_privilege(delete_current_privilege: bool = True):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)